### ISSUE TITLE
default item sorting in preferences

### ISSUE THREAD

**Issue by liomj** [XoopsModules25x/modulebuilder#162]
if this is doable i like to suggest add option item sorting in preferences of generated module

 we can define in modulebuilder parameter list which field can be sort and automatically add it in preferences of our module

example sort by date, name,weight etc

**Comment by ggoffy** [comment]
hi thank you for your suggestion, but I do not think that this make sense:
make preferences for sorting

- in blocks?
- for tables in admin area?
- for index file in user area?
- for detail file in user area?
- sorting by one field or multiple fields?
- ....

IMO this would blow up the module and especially the preferences

**Comment by liomj** [comment]
yes maybe next time. I see some module have this option like publisher implementing the sort order of the items throughout the module. module-wide  

![publisher](https://user-images.githubusercontent.com/16532279/193496500-63e392c0-399d-439a-9c3c-e21f50186d05.jpg)
